`globalVerbosity` from README is not mutable

The statement in the README seems to be outdated:

```
// Update verbosity
system.globalVerbosity = Verbosity.DEBUG; // Global parameter
```

It's not mutable anymore:

```
    sources/contract.spec.ts:10:16 - error TS2540: Cannot assign to 'globalVerbosity' because it is a read-only property.

    10         system.globalVerbosity = Verbosity.DEBUG; // Global parameter
 ```

--- a/README.md
+++ b/README.md
@@ -22,7 +22,6 @@ let treasure = await system.treasure('my-treasure');
 system.name(treasure, 'my treasure'); // Set a name for the contract. Useful for snapshot testing.
 
 // Update verbosity
-system.globalVerbosity = Verbosity.DEBUG; // Global parameter
 system.verbosity(treasure, Verbosity.ERROR); // Overwrite for a specific contract
 
 // Track contract transactions and events
@@ -31,7 +30,7 @@ let tracker = system.track(treasure);
 // Logger to collect VM logs from a contract
 let logger = system.log(treasure);
 
-// Create a random unknown address that would be treated as unititialized contract
+// Create a random unknown address that would be treated as uninitialized contract
 let unknownAddress = randomAddress('some-unknown-seed'); // This seed is used to generate deterministic address
 
 // Send an empty message to the unknown address

